client/src/components/ClickableBox.tsx

```diff
@@ -5,13 +5,18 @@ import {useAppSelector} from "../store/hooks";
 import {FavoriteIcon} from "./icons/FavoriteIcon";
 import {selectGender} from "../store/user-details/user-details.selectors";
 import {Gender} from "../store/user-details/user-details.model";
+import {DeleteIcon} from "./icons/DeleteIcon";
+import {EditIcon} from "./icons/EditIcon";
 
 interface ClickableBoxProps {
     name: string,
     id: string,
     imageUrl: string;
     onPress?: () => void;
     hasIcon?: boolean;
+    editMode?: boolean;
```

Comment:
Keep standardization everywhere.
I saw `editMode`, `isEditMode`, `editRemoveMode`... I'd call it `isEditMode` - even though delete is included, think of it as adjusting the screen as you like by editing/removing items.

Reply:
done